src/metabase/query_processor/pivot.clj

```diff
@@ -96,7 +96,8 @@
                        :pivot-cols    pivot-cols}))))
   (sort-by
    (partial group-bitmask num-breakouts)
-   (distinct
+   (m/distinct-by
```

Comment:
This change was needed because some group bitmasks end up being equivalent [0 1] and [1 0], but the order still matters as they correspond to how the user set up the pivot-rows and/or pivot-cols, which affects how the result is displayed. But, prior to this change, [0 1] and [1 0] would not be considered equivalent, and this would result in duplicated rows for the given group.